includes/admin/customers/class-customer-addresses-table.php

```diff
@@ -210,18 +210,21 @@ public function column_customer( $item ) {
 	 * @access public
 	 * @since 3.0
 	 *
-	 * @param array Address
+	 * @param array $item Address object.
 	 *
 	 * @return string Displays a checkbox
 	 */
 	public function column_cb( $item ) {
 		$customer = edd_get_customer_by( 'id', $item['customer_id'] );
-		$name     = isset( $customer->name ) ? $customer->name : $customer->email;
+		$name     = sprintf( __( 'Address ID: ' ) . $item['id'] );
+		if ( ! empty( $customer->name ) ) {
+			$name = $customer->name ? $customer->name : $customer->email;
```

Comment:
I think this no longer needs to be a ternary, just the assignment since it is within a condition (and then the email can be removed from the comment) ... because if the `$customer->name` is not empty above then this check is redundant.